.travis.yml

```diff
@@ -10,10 +10,11 @@ env:
   - PIPENV_IGNORE_VIRTUALENVS=1
 
 install:
-  - pip install pipenv
+  - pip install pipenv black
```

Comment:
I would prefer to add `black` to dev dependencies in Pipfile

Reply:
I tried, we can't. There are only pre-releases available. pipenv doesn't like it much.

Reply:
`pipenv install --dev "black==19.10b0"` does work for me

Reply:
Interesting! Let's try again then.